### ISSUE TITLE
About Speech Enhancement scores on SUPERB leaderboard

### ISSUE THREAD

**Issue by leo19941227** [Speech-Lab-IITM/CCC-wav2vec-2.0#1]
Hi!

Thanks for the great repo!
I find that CCC-wav2vec 2.0 performs especially well on SUPERB SE task, surpassing WavLM Large with a large margin.
I am trying to reproduce it but I not yet successfully get the similar score. (PESQ around 3)
May I ask how did you fine-tune CCC-wav2vec 2.0 on SUPERB SE?
Specifically, what are the tuning hyper-parameters? (e.g. batch size and learning rate)
Thanks!

**Comment by leo19941227** [comment]
P.s. I tried the default batch size and learning rate 1e-4 and 5e-5, and adding `--upstream_feature_normalize` to apply layer norm on the representation, the best result I get is around 2.6 PESQ.
